plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -61,6 +61,9 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_ARRAY:
                 return false;
 
+            case SUPPORTS_TOPN_PUSHDOWN:
+                return false;
```

Comment:
I did not count. But maybe it should be `false` by default?

Reply:
It's going to turn `true` by default soon I think.